lms/static/scripts/frontend_apps/utils/dashboard/organization-name.ts

```diff
@@ -0,0 +1,5 @@
+import type { DashboardConfig } from '../../config';
+
+export function organizationName(dashboardConfig: DashboardConfig): string {
```

Comment:
"Organization name" is a slightly confusing name here I think, because "All courses" is definitely not an organization name. Instead this is the title for the root or home view in the dashboard.

Reply:
Yeah, good point. Let me change ot.